### PR TITLE
General Cleanup

### PR DESCRIPTION
**Pull request by bergieboy**
- Fix Linter Errors
- Fix Flow Errors
- Switch to LUK components where possible
- Switch to LUK Colors

### CODE REVIEW

File: src/core/tracking/google/GoogleAnalytics.js [hunk 1]
```diff
@@ -4,6 +4,6 @@
 
 const GOOGLE_TRACKING_ID :string = 'UA-118446829-4';
 
-export {
+export default {
```
Comment (UnsungHero97) [inline src/core/tracking/google/GoogleAnalytics.js:7]: Not necessary.

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

File: src/components/controls/PlusMinus.js [hunk 2]
```diff
@@ -25,70 +21,21 @@ const Wrapper = styled.div`
...
-
-  &:focus {
-    outline: none;
-  }
-`;
-
 const PlusMinus = ({ value, onChange } :Props) => (
   <Wrapper>
-    <Button disabled={!value} onClick={() => onChange(value - 1)}>
+    <IconButton disabled={!value} aria-labelledby="numberOfInfants" onClick={() => onChange(value - 1)}>
```
Comment (solomontang) [inline src/components/controls/PlusMinus.js:32]: Isn't the icon button round where as the one it's replacing is rectangular?

File: src/components/controls/StyledInput.js [hunk 3]
```diff
@@ -7,16 +7,16 @@ import styled, { css } from 'styled-components';
...
   border-radius: 3px;
   background-color: #f9f9fd;
   border: solid 1px #dcdce7;
   color: #2e2e34;
   padding: 12px 20px;
-  margin-bottom: ${props => (props.padBottom ? 20 : 0)}px;
+  margin-bottom: ${(props) => (props.padBottom ? 20 : 0)}px;
 
   &:focus {
     box-shadow: inset 0 0 0 1px rebeccapurple;
```
Comment (solomontang) [inline src/components/controls/StyledInput.js:22]: judging by the color used, this definitely feels like i put it here... 🤔

File: src/components/controls/StyledInput.js [hunk 4]
```diff
@@ -68,8 +67,8 @@ const SearchIcon = styled(FontAwesomeIcon).attrs({
   top: 0;
   bottom: 0;
 `;
-
-export const SearchInput = props => (
+/* eslint-disable */
```
[the remark this answers is not in the document]
Comment (solomontang) [reply src/components/controls/StyledInput.js:70]: prop spreading maybe?

File: src/components/controls/helpers/LatticeLuxonUtils.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 // @flow
 import LuxonUtils from '@date-io/luxon';
 import type { DateTime, DurationObject } from 'luxon';
-
+/* eslint-disable class-methods-use-this */
```
[the remark this answers is not in the document]
Comment (solomontang) [reply src/components/controls/helpers/LatticeLuxonUtils.js:4]: LatticeLuxonUtils should be provided by LUK. this file can prob just go away.

File: src/core/tracking/google/GoogleAnalytics.js (hunk 1 above)
Comment (bergieboy) [reply src/core/tracking/google/GoogleAnalytics.js:7]: linter says 'export default' is preferred.

File: src/containers/location/providers/ActiveProviderLocationLayer.js [hunk 6]
```diff
@@ -14,10 +14,12 @@ const layout = { 'icon-image': 'activeMapPin' };
 
 type Props = {
   providerLocations :List;
-  onFeatureClick ? :(data, feature) => void;
+  onFeatureClick ?:(location :Map, feature :{}) => void;
 };
 
 const ActiveProviderLocationLayer = (props :Props) => (
+  /* eslint-disable */
```
[the remark this answers is not in the document]
Comment (solomontang) [reply src/containers/location/providers/ActiveProviderLocationLayer.js:21]: still looks like prop spreading to me

File: src/components/controls/StyledInput.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (bergieboy) [reply src/components/controls/StyledInput.js:70]: This file is being deleted.

File: src/components/controls/StyledInput.js [hunk 7]
```diff
@@ -46,7 +46,6 @@ export const StyledTextArea = styled.textarea`
 
 export default StyledInput;
```
[the remark this answers is not in the document]
Comment (bergieboy) [reply src/components/controls/StyledInput.js:47]: This file is being deleted.

File: src/components/controls/helpers/LatticeLuxonUtils.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (bergieboy) [reply src/components/controls/helpers/LatticeLuxonUtils.js:4]: Screen Shot 2020-10-27 at 3 13 58 PM

File: src/containers/about/AboutPage.js [hunk 8]
```diff
@@ -129,7 +124,7 @@ class AboutPage extends React.Component {
       <LogoRow>
         {logoUrls.map((url) => (
           <LogoLink href={url} key={url} width={width} height={IMG_HEIGHT[url] || 40}>
-            <img src={URL_TO_IMG[url]} />
+            <img alt={URL_TO_IMG[url]} src={URL_TO_IMG[url]} />
```
[the remark this answers is not in the document]
Comment (bergieboy) [reply src/containers/about/AboutPage.js:127]: yes, my bad. need to create a new object for alt strings.

File: src/containers/location/providers/ProviderHeaderContainer.js [hunk 9]
```diff
@@ -160,17 +152,16 @@ class ProviderHeaderContainer extends React.Component {
...
     const city = getValue(provider, PROPERTY_TYPES.CITY);
-    const zip = getValue(provider, PROPERTY_TYPES.ZIP);
 
-    const isPopUp = getValue(provider, PROPERTY_TYPES.IS_POP_UP);
+    // const status = getValues(provider, PROPERTY_TYPES.STATUS);
+    // const paymentOptions = getValues(provider, PROPERTY_TYPES.PAYMENT_OPTIONS);
+    // const url = getValue(provider, PROPERTY_TYPES.URL);
+    // const street = getValue(provider, PROPERTY_TYPES.ADDRESS);
+    // const zip = getValue(provider, PROPERTY_TYPES.ZIP);
+    // const isPopUp = getValue(provider, PROPERTY_TYPES.IS_POP_UP);
```
[the remark this answers is not in the document]
Comment (bergieboy) [reply src/containers/location/providers/ProviderHeaderContainer.js:164]: Was going to keep it here until I did all the other updates, but I'll just delete it.

File: src/containers/location/providers/ProviderPopup.js [hunk 10]
```diff
@@ -85,20 +85,17 @@ const ProviderPopup = ({
...
   const city = getValue(provider, PROPERTY_TYPES.CITY);
-  const zip = getValue(provider, PROPERTY_TYPES.ZIP);
-
-  const isOperating = getValue(provider, PROPERTY_TYPES.STATUS) === FACILITY_STATUSES.OPEN;
+  // const isOperating = getValue(provider, PROPERTY_TYPES.STATUS) === FACILITY_STATUSES.OPEN;
+  // const providerEKID = provider.getIn([OPENLATTICE_ID_FQN, 0]);
+  // const status = getValues(provider, PROPERTY_TYPES.STATUS);
+  // const street = getValue(provider, PROPERTY_TYPES.ADDRESS);
+  // const url = getValue(provider, PROPERTY_TYPES.URL);
+  // const zip = getValue(provider, PROPERTY_TYPES.ZIP);
```
[the remark this answers is not in the document]
Comment (bergieboy) [reply src/containers/location/providers/ProviderPopup.js:98]: Was going to keep it here until I did all the other updates, but I'll just delete it.

File: src/components/controls/StyledInput.js (hunk 3 above)
Comment (bergieboy) [reply src/components/controls/StyledInput.js:22]: this is going away

File: src/components/controls/PlusMinus.js (hunk 2 above)
Comment (bergieboy) [reply src/components/controls/PlusMinus.js:32]: The wrapper is the same color and there is no border on the icon button. So it appears circular on hover, but it's not offensive to me.

File: src/core/tracking/google/GoogleAnalytics.js (hunk 1 above)
Comment (UnsungHero97) [reply src/core/tracking/google/GoogleAnalytics.js:7]: It's a warning, not an error.

**Review by UnsungHero97 — APPROVED** [review]

**Review by solomontang — APPROVED** [review]

**Pull request merged**